README.md

```diff
@@ -443,6 +454,94 @@ cd lib/user-interface/react/
 npm run dev
 ```
 
+## Usage and Features
+
+### OpenAI Specification Compatibility
+
+We now provide greater support for the [OpenAI specification](https://platform.openai.com/docs/api-reference) for model inference and embeddings.
+We utilize LiteLLM as a proxy for both models we spin up on behalf of the user and additional models configured through the config.yaml file, and because of that, the
+LISA REST API endpoint allows for a central location for making text generation and embeddings requests. We do not support the entire API specification, but most notably
+we do support the following APIs, subject to model and container compatibility:
+
+- /models
+- /chat/completions
+- /completions
+- /embeddings
+
+By supporting the OpenAI spec, we can more easily allow users to integrate their collection of models into their LLM applications and workflows. In LISA, users can authenticate
+using their OpenID Connect Identity Provider, or with an API token created through the DynamoDB token workflow as described [here](#programmatic-api-tokens). Once the token
+is retrieved, users can use that in direct requests to the LISA Serve REST API. If using the IdP, users must set the 'Authorization' header, otherwise if using the API token,
+users must set the 'Api-Key' header. After that, requests to `https://${lisa_serve_alb}/v2/serve` will handle the OpenAI API calls. As an example, the following call can list all
+models that LISA is aware of, assuming usage of the API token.
+
+```shell
+curl -s -H 'Api-Key: your-api-token' -X GET https://${lisa_serve_alb}/v2/serve/models
+```
+
+If using the IdP, the request would look like the following:
+
+```shell
+curl -s -H 'Authorization: Bearer your-bearer-token' -X GET https://${lisa_serve_alb}/v2/serve/models
+```
+
+When using a library that requests an OpenAI-compatible base_url, you can provide `https://${lisa_serve_alb}/v2/serve` here. All of the OpenAI routes will
```

Comment:
Just want to make sure `lisa_serve_alb` is still corect with the ALB complications we had.

Reply:
Yes, these requests still go to the load balancer and not the APIGW endpoint